### PR TITLE
Refs bug #391662 Unify notification

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets

- [#391662](https://dev.sun-asterisk.com/issues/391662)

## What's this PR do ?

- [x] Update using a same notification type

## Library
*(List package, library third party add new include version)*

- N/A

## Performance

- [ ] Resolved n + 1 query
- [ ] Run explain query already
- [ ] Time run rake task : 1000 ms

## Checklist

- [x] It was tested in local success?
- [x] Fill link PR into ticket and the opposite
- [x] Note purpose/reason, solution, scope of influence into ticket

## ENV note

```
```

## Notes
*(Other notes)*

### CODE REVIEW

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>failure</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr></table>

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>failure</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr></table>

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>failure</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr></table>

File: app/Http/Controllers/MessageHistoryController.php [hunk 1]
```diff
@@ -29,9 +29,15 @@ public function destroy(MessageHistory $message)
             $this->messageHistoryRepository->delete($message->id);
 
             return redirect(route('history.show', ['history' => $payloadHistoryId]))
-                    ->with('messageSuccess', 'This message history successfully deleted');
+                ->with('messageSuccess', [
+                    'status' => 'Delete success',
+                    'message' => 'This message history successfully deleted'
```
Comment (framgiaciv3) [inline app/Http/Controllers/MessageHistoryController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/MessageHistoryController.php</br>**Line**: 34</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/MessageHistoryController.php [hunk 2]
```diff
@@ -29,9 +29,15 @@ public function destroy(MessageHistory $message)
...
+                ]);
         } catch (Exception $exception) {
-            return redirect()->back()->with('messageFail', 'Delete failed. Something went wrong');
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Delete failed',
+                'message' => 'Delete failed. Something went wrong'
```
Comment (framgiaciv3) [inline app/Http/Controllers/MessageHistoryController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/MessageHistoryController.php</br>**Line**: 39</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/PayloadController.php [hunk 3]
```diff
@@ -73,7 +73,10 @@ public function store(PayloadCreateRequest $request, $webhookId)
             }
 
             DB::commit();
-            $request->session()->flash('messageSuccess', 'This payload successfully created');
+            $request->session()->flash('messageSuccess', [
+                'status' => 'Create success',
+                'message' => 'This payload successfully created'
```
Comment (framgiaciv3) [inline app/Http/Controllers/PayloadController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadController.php</br>**Line**: 78</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/PayloadController.php [hunk 4]
```diff
@@ -124,7 +127,10 @@ public function update(PayloadUpdateRequest $request, Webhook $webhook, Payload
             }
 
             DB::commit();
-            $request->session()->flash('messageSuccess', 'This payload successfully updated');
+            $request->session()->flash('messageSuccess', [
+                'status' => 'Update success',
+                'message' => 'This payload successfully updated'
```
Comment (framgiaciv3) [inline app/Http/Controllers/PayloadController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadController.php</br>**Line**: 132</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/PayloadController.php [hunk 5]
```diff
@@ -143,16 +149,25 @@ public function destroy(Webhook $webhook, Payload $payload)
         $this->authorize('delete', [$payload, $webhook]);
         if ($payload->conditions->count() > 0) {
             return redirect()->back()
-                ->with('messageFail', 'This payload has some conditions to be related with, please delete them first');
+                ->with('messageFail', [
+                    'status' => 'Delete failed',
+                    'message' => 'This payload has some conditions to be related with, please delete them first'
```
Comment (framgiaciv3) [inline app/Http/Controllers/PayloadController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadController.php</br>**Line**: 154</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/PayloadController.php [hunk 6]
```diff
@@ -143,16 +149,25 @@ public function destroy(Webhook $webhook, Payload $payload)
...
         }
 
         try {
             $this->payloadRepository->delete($payload->id);
 
             return redirect()->route('webhooks.edit', $webhook)
-                             ->with('messageSuccess', 'This payload successfully deleted');
+                ->with('messageSuccess', [
+                    'status' => 'Delete success',
+                    'message' => 'This payload successfully deleted'
```
Comment (framgiaciv3) [inline app/Http/Controllers/PayloadController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadController.php</br>**Line**: 164</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/PayloadController.php [hunk 7]
```diff
@@ -143,16 +149,25 @@ public function destroy(Webhook $webhook, Payload $payload)
...
+                ]);
         } catch (Exception $exception) {
-            return redirect()->back()->with('messageFail', 'Delete failed. Something went wrong');
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Delete failed',
+                'message' => 'Delete failed. Something went wrong'
```
Comment (framgiaciv3) [inline app/Http/Controllers/PayloadController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadController.php</br>**Line**: 169</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/BotController.php [hunk 8]
```diff
@@ -33,9 +33,15 @@ public function destroy(Bot $bot)
         try {
             $this->botRepository->delete($bot->id);
 
-            return redirect('/bots')->with('messageSuccess', __('message.bot.notification.delete.success'));
+            return redirect('/bots')->with('messageSuccess', [
+                'status' => 'Delete success',
+                'message' => __('message.bot.notification.delete.success')
```
Comment (framgiaciv3) [inline app/Http/Controllers/BotController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/BotController.php</br>**Line**: 38</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/BotController.php [hunk 9]
```diff
@@ -33,9 +33,15 @@ public function destroy(Bot $bot)
...
+            ]);
         } catch (Exception $exception) {
-            return redirect()->back()->with('messageFail', __('message.bot.notification.delete.fail'));
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Delete failed',
+                'message' => __('message.bot.notification.delete.fail')
```
Comment (framgiaciv3) [inline app/Http/Controllers/BotController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/BotController.php</br>**Line**: 43</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/BotController.php [hunk 10]
```diff
@@ -52,9 +58,15 @@ public function store(BotCreateRequest $request)
         try {
             $bot = $this->botRepository->create($data);
             return redirect()->route('bots.edit', $bot)
-                             ->with('messageSuccess', 'This bot successfully created');
+                ->with('messageSuccess', [
+                    'status' => 'Create success',
+                    'message' => 'This bot successfully created'
```
Comment (framgiaciv3) [inline app/Http/Controllers/BotController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/BotController.php</br>**Line**: 63</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/BotController.php [hunk 11]
```diff
@@ -52,9 +58,15 @@ public function store(BotCreateRequest $request)
...
+                ]);
         } catch (QueryException $exception) {
-            return redirect()->back()->with('messageFail', 'Create failed. Something went wrong')->withInput();
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Create failed',
+                'message' => 'Create failed. Something went wrong'
```
Comment (framgiaciv3) [inline app/Http/Controllers/BotController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/BotController.php</br>**Line**: 68</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/BotController.php [hunk 12]
```diff
@@ -73,9 +85,15 @@ public function update(BotUpdateRequest $request, Bot $bot)
         try {
             $bot = $this->botRepository->update($bot->id, $data);
             return redirect()->route('bots.edit', $bot)
-                             ->with('messageSuccess', 'This bot successfully updated');
+                ->with('messageSuccess', [
+                    'status' => 'Update success',
+                    'message' => 'This bot successfully updated'
```
Comment (framgiaciv3) [inline app/Http/Controllers/BotController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/BotController.php</br>**Line**: 90</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/BotController.php [hunk 13]
```diff
@@ -73,9 +85,15 @@ public function update(BotUpdateRequest $request, Bot $bot)
...
+                ]);
         } catch (QueryException $exception) {
-            return redirect()->back()->with('messageFail', 'Update failed. Something went wrong')->withInput();
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Update failed',
+                'message' => 'Update failed. Something went wrong'
```
Comment (framgiaciv3) [inline app/Http/Controllers/BotController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/BotController.php</br>**Line**: 95</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/PayloadHistoryController.php [hunk 14]
```diff
@@ -73,9 +73,15 @@ public function destroy(PayloadHistory $history)
             $this->payloadHistoryRepository->delete($history->id);
 
             return redirect(route('history.index'))
-                    ->with('messageSuccess', 'This payload history successfully deleted');
+                ->with('messageSuccess', [
+                    'status' => 'Delete success',
+                    'message' => 'This payload history successfully deleted'
```
Comment (framgiaciv3) [inline app/Http/Controllers/PayloadHistoryController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadHistoryController.php</br>**Line**: 78</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/PayloadHistoryController.php [hunk 15]
```diff
@@ -73,9 +73,15 @@ public function destroy(PayloadHistory $history)
...
+                ]);
         } catch (Exception $exception) {
-            return redirect()->back()->with('messageFail', 'Delete failed. Something went wrong');
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Delete failed',
+                'message' => 'Delete failed. Something went wrong'
```
Comment (framgiaciv3) [inline app/Http/Controllers/PayloadHistoryController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadHistoryController.php</br>**Line**: 83</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/WebhookController.php [hunk 16]
```diff
@@ -62,9 +62,15 @@ public function store(WebhookCreateRequest $request)
             $webhook = $this->webhookRepository->create($data);
 
             return redirect()->route('webhooks.edit', $webhook)
-                ->with('messageSuccess', 'This webhook successfully created');
+                ->with('messageSuccess', [
+                    'status' => 'Create success',
+                    'message' => 'This webhook successfully created'
```
Comment (framgiaciv3) [inline app/Http/Controllers/WebhookController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 67</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/WebhookController.php [hunk 17]
```diff
@@ -62,9 +62,15 @@ public function store(WebhookCreateRequest $request)
...
+                ]);
         } catch (QueryException $exception) {
-            return redirect()->back()->with('messageFail', 'Create failed. Something went wrong')->withInput();
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Create success',
+                'message' => 'Create failed. Something went wrong'
```
Comment (framgiaciv3) [inline app/Http/Controllers/WebhookController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 72</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/WebhookController.php [hunk 18]
```diff
@@ -99,9 +105,15 @@ public function update(WebhookUpdateRequest $request, Webhook $webhook)
         try {
             $webhook = $this->webhookRepository->update($webhook->id, $data);
             return redirect()->route('webhooks.edit', $webhook)
-                ->with('messageSuccess', 'This webhook successfully updated');
+                ->with('messageSuccess', [
+                    'status' => 'Update success',
+                    'message' => 'This webhook successfully updated'
```
Comment (framgiaciv3) [inline app/Http/Controllers/WebhookController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 110</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/WebhookController.php [hunk 19]
```diff
@@ -99,9 +105,15 @@ public function update(WebhookUpdateRequest $request, Webhook $webhook)
...
+                ]);
         } catch (QueryException $exception) {
-            return redirect()->back()->with('messageFail', 'Update failed. Something went wrong')->withInput();
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Update failed',
+                'message' => 'Update failed. Something went wrong'
```
Comment (framgiaciv3) [inline app/Http/Controllers/WebhookController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 115</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/WebhookController.php [hunk 20]
```diff
@@ -116,27 +128,39 @@ public function destroy(Webhook $webhook)
         $this->authorize('delete', $webhook);
         if ($webhook->payloads->count() > 0) {
             return redirect()->back()
-                ->with('messageFail', 'This webhook has some payloads to be related with, please delete them first');
+                ->with('messageFail', [
+                    'status' => 'Delete failed',
+                    'message' => 'This webhook has some payloads to be related with, please delete them first'
```
Comment (framgiaciv3) [inline app/Http/Controllers/WebhookController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 133</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/WebhookController.php [hunk 21]
```diff
@@ -116,27 +128,39 @@ public function destroy(Webhook $webhook)
...
+                ]);
         }
 
         if ($webhook->mappings->count() > 0) {
             return redirect()->back()
-                ->with('messageFail', 'This webhook has some mappings to be related with, please delete them first');
+                ->with('messageFail', [
+                    'status' => 'Delete failed',
+                    'message' => 'This webhook has some mappings to be related with, please delete them first'
```
Comment (framgiaciv3) [inline app/Http/Controllers/WebhookController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 141</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/WebhookController.php [hunk 22]
```diff
@@ -116,27 +128,39 @@ public function destroy(Webhook $webhook)
...
 
         if ($webhook->payloadHistories->count() > 0) {
             return redirect()->back()
-                ->with(
-                    'messageFail',
-                    'This webhook has some payload histories to be related with, please delete them first'
-                );
+                ->with('messageFail', [
+                    'status' => 'Delete failed',
+                    'message' => 'This webhook has some payload histories to be related with, please delete them first'
```
Comment (framgiaciv3) [inline app/Http/Controllers/WebhookController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 149</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/WebhookController.php [hunk 23]
```diff
@@ -116,27 +128,39 @@ public function destroy(Webhook $webhook)
...
+                ]);
         }
 
         try {
             $this->webhookRepository->delete($webhook->id);
-            return redirect('/webhooks')->with('messageSuccess', 'This webhook successfully deleted');
+            return redirect('/webhooks')->with('messageSuccess', [
+                'status' => 'Delete success',
+                'message' => 'This webhook successfully deleted'
```
Comment (framgiaciv3) [inline app/Http/Controllers/WebhookController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 157</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/WebhookController.php [hunk 24]
```diff
@@ -116,27 +128,39 @@ public function destroy(Webhook $webhook)
...
+            ]);
         } catch (Exception $exception) {
-            return redirect()->back()->with('messageFail', 'Delete failed. Something went wrong');
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Delete failed',
+                'message' => 'Delete failed. Something went wrong'
```
Comment (framgiaciv3) [inline app/Http/Controllers/WebhookController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 162</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/UserController.php [hunk 25]
```diff
@@ -54,9 +54,15 @@ public function store(UserCreateRequest $request)
                 $message->to($email)->subject('Welcome to Chatwork Forwarder Application');
             });
             return redirect()->route('users.edit', $user)
-                             ->with('messageSuccess', 'This user successfully created');
+                ->with('messageSuccess', [
+                    'status' => 'Create success',
+                    'message' => 'This user successfully created'
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 59</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/UserController.php [hunk 26]
```diff
@@ -54,9 +54,15 @@ public function store(UserCreateRequest $request)
...
+                ]);
         } catch (QueryException $exception) {
-            return redirect()->back()->with('messageFail', 'Create failed. Something went wrong')->withInput();
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Create success',
+                'message' => 'Create failed. Something went wrong'
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 64</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/UserController.php [hunk 27]
```diff
@@ -96,12 +102,18 @@ public function update(UserUpdateRequest $request, $id)
 
             return response()->json([
                 'error' => false,
-                'messageSuccess' => 'This user successfully updated',
+                'messageSuccess' => [
+                    'status' => 'Update success',
+                    'message' => 'This user successfully updated'
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 107</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/UserController.php [hunk 28]
```diff
@@ -96,12 +102,18 @@ public function update(UserUpdateRequest $request, $id)
...
+                ],
             ]);
         } catch (QueryException $exception) {
             return response()->json([
                 'error' => true,
-                'messageFail' => 'Update failed. Something went wrong',
+                'messageFail' => [
+                    'status' => 'Update success',
+                    'message' => 'Update failed. Something went wrong'
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 115</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/UserController.php [hunk 29]
```diff
@@ -116,13 +128,22 @@ public function destroy(User $user)
     {
         try {
             if ($user->id == Auth::user()->id) {
-                return redirect()->back()->with('messageFail', 'Delete failed, Cannot delete myself');
+                return redirect()->back()->with('messageFail', [
+                    'status' => 'Delete failed',
+                    'message' => 'Delete failed, Cannot delete myself'
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 133</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/UserController.php [hunk 30]
```diff
@@ -116,13 +128,22 @@ public function destroy(User $user)
...
+                ]);
             } else {
                 $this->userRepository->delete($user->id);
-                return redirect('/admin/users')->with('messageSuccess', 'This user successfully deleted');
+                return redirect('/admin/users')->with('messageSuccess', [
+                    'status' => 'Delete success',
+                    'message' => 'This user successfully deleted'
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 139</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/UserController.php [hunk 31]
```diff
@@ -116,13 +128,22 @@ public function destroy(User $user)
...
+                ]);
             }
         } catch (Exception $exception) {
-            return redirect()->back()->with('messageFail', 'Delete failed. Something went wrong');
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Delete failed',
+                'message' => 'Delete failed. Something went wrong'
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 145</br>Each line in an array declaration must end in a comma

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr></table>

**Review by nguyenthinhhn — APPROVED** [review]

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr></table>

**Review by ghost — APPROVED** [review]

**Review by quanghung97 — APPROVED** [review]
OK

**Comment by phucqd-1305** [comment]
OK

**Pull request merged**